src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,64 @@
+.watch {
+  @include page-center;
+  @include watch-size;
+  border: 5px solid gray;
+  border-radius: 50%;
+
+  &__hand {
+    @include page-center;
+    animation-name: hands-movement;
+    animation-iteration-count: infinite;
+    animation-timing-function: steps(60, end);
+    transform-origin: center left;
+
+    @keyframes hands-movement {
+      0% {
+        transform: translate(0, -50%) rotate(-90deg);
+      }
+
+      100% {
+        transform: translate(0, -50%) rotate(270deg);
+      }
+    }
+
+    &--minute {
+      @include minute-hand-size;
+      background-color: $main-color;
+      animation-duration: 3600s;
+    }
+
+    &--second {
+      @include second-hand-size;
```

Comment:
If that's not what you mean - you want it to be called like `watch-hand-minutes` or sth 😄 

![image](https://user-images.githubusercontent.com/103033758/235628524-8e847c91-748b-4b6c-9048-f572e610de74.png)

Reply:
😁